config/template.py

```diff
@@ -0,0 +1,50 @@
+import os
+import binascii
+
+def render_template(template, destination, **kwargs):
+
+
+   here = os.path.abspath(os.path.dirname(__file__))
+   template = os.path.join(here, 'kinto.tpl') 
+   destination = os.path.join(here, 'kinto.ini')
+
+    with open(template, 'r') as f:
+        raw_template = f.read()
+       
+        rendered = raw_template.format(**kwargs)
+
+        with open(destination, 'w+') as output:
+            output.write(rendered)
+
+
+def init():
+	values= {}
+	values['secret'] = binascii.b2a_hex(os.urandom(32))
+
+	backend = raw_input("Which backend to use ? (1 - in-memory, 2 - postgresql, 3 - redis) :")
```

Comment:
Please use ``six.moves.input`` to make this code compatible with both Python2 and Python3.

http://pythonhosted.org/six/#module-six.moves